tests/bluetooth/host/buf/bt_buf_get_cmd_complete/testcase.yaml

```diff
@@ -0,0 +1,14 @@
+common:
+    tags: test_framework bluetooth host
+tests:
+  bluetooth.host.bt_buf_get_cmd_complete.default:
+     type: unit
+  bluetooth.host.bt_buf_get_cmd_complete.hci_acl_flow_control:
+     type: unit
+     extra_args: COMPILE_SWITCH_PARAM=CONFIG_BT_HCI_ACL_FLOW_CONTROL
```

Comment:
I already tried your suggested way and seems not be working (at least when the test type is set to unit testing using `type: unit`)

Reply:
> I already tried your suggested way and it seems not be working

yes, because Kconfig is not supported, which is why I also wrote :wink: : 
> What is done in this PR is to invent a new config scheme instead of supporting Kconfig.
> 
> I will take a look to see if I can get Kconfig supported with unittest so that both this and the comments in https://github.com/zephyrproject-rtos/zephyr/pull/40073 can be cleaned up.

Reply:
just made the first comment so we remember what to clean up and not forgetting.
Little you can do atm.